components/map.tsx

```diff
@@ -8,6 +10,12 @@ export default function Page() {
         defaultCenter={{ lat: 30.2672, lng: -97.7393 }}
         defaultZoom={10}
       />
```

Comment:
One last thing I forgot to mention: We'll need to put the `Marker`s inside of the `<GoogleMapReact>` component. Something like this should work:

```tsx
<GoogleMapReact
  bootstrapURLKeys={{ key: process.env.NEXT_PUBLIC_GOOGLE_MAPS_KEY }}
  defaultCenter={{ lat: 30.2672, lng: -97.7393 }}
  defaultZoom={10}
>
  <Marker
    lat={30.2672}
    lng={-97.7393}
    text="Crag Climbing 🧗🏾‍♂️"
    color="blue"
  />
</GoogleMapReact>
```

Reply:
Ahhh sickkkkk sounds good man

Reply:
Bro I got it to render! It is showing the text and the emoji.  Now we just need to tidy up the style of it but this works.  We got the map marker down.